templates/types/streaming/nextjs/package.json

```diff
@@ -30,8 +30,8 @@
     "llamaindex": "0.8.2",
     "lucide-react": "^0.294.0",
     "next": "^15.0.3",
-    "react": "19.0.0-rc-5c56b873-20241107",
-    "react-dom": "19.0.0-rc-5c56b873-20241107",
+    "react": "^18.2.0",
```

Comment:
@thucpn @leehuwuj  react 18.2 is working with nextjs 15?

Reply:
I merged with latest main, looks like it's fixed.